react/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import BookList from './components/BookList';
+import BookFilters from './components/BookFilters';
+import './App.css';
+import 'roboto-npm-webfont';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {filterType: 0, filterText: ""};
```

Comment:
I think that `0` should belong to a constant

Reply:
Where should i define that constant?